app/assets/stylesheets/modules/_header.scss

```diff
@@ -11,7 +11,15 @@ header {
   h1 {
     font-size: 55px;
     line-height: 1;
-    color: #fefeff;
+    color: $header_title_color;
```

Comment:
Name of variable `header_title_color` should be written in all lowercase letters with hyphens instead of underscores